apple/internal/apple_framework_import.bzl

```diff
@@ -59,6 +59,15 @@ def _is_swiftmodule(path):
     """Predicate to identify Swift modules/interfaces."""
     return path.endswith(".swiftmodule") or path.endswith(".swiftinterface")
 
+def _swiftmodules_for_cpu(swiftmodule_files, cpu):
+    """Select the cpu specific swiftmodule."""
+    for swiftmodule in swiftmodule_files:
+        root, _ = paths.split_extension(swiftmodule.basename)
+        if root == cpu:
```

Comment:
I think this might have been mentioned elsewhere but does the CPU correctly handle the armv7 case? I think in that case the swiftmodule is `arm.swiftmodule` 🙃  https://github.com/ios-bazel-users/ios-bazel-users/blob/master/swift_static_framework/swift_static_framework.bzl#L8-L13

Reply:
Thanks, will update.

Reply:
`arm` is supported in c4ab505895a5cf37da737b8cd62b56084596c74e